zebra-chain/Cargo.toml

```diff
@@ -132,8 +131,7 @@ serde_json = { version = "1.0.113", optional = true }
 # Production feature async-error and testing feature proptest-impl
 tokio = { version = "1.36.0", optional = true }
 
-# Production feature getblocktemplate-rpcs
-zcash_address = { version = "0.3.1", optional = true }
+zcash_address = { version = "0.3.1" }
```

Comment:
Note: This is now used in the `From<Network>` impl for `zcash_primitives::consensus::Network` outside the `getblocktemplate-rpcs` feature.